server/src/components/ListItem.js

```diff
@@ -121,10 +68,15 @@ export default class ListItem extends Component {
               </div>
 
               <div>
+                <button
+                  onClick={() => togglePinned(id)}
+                  className={`btn btn-sm tooltipped tooltipped-s ${pinned && 'text-blue'}`}
+                  aria-label="Pin this delivery"
+                ><PinIcon /></button>
```

Comment:
Minor nit, but this spacing looks weird to me. Maybe like this?

```jsx
<button
  onClick={() => togglePinned(id)}
  className={`btn btn-sm tooltipped tooltipped-s ${pinned && 'text-blue'}`}
  aria-label="Pin this delivery">
    <PinIcon />
</button>
```

Reply:
Is that spacing coming from linter settings? Consistency is all I really care about.

Reply:
Yeah, it yells at me if I do your above spacing ☹️

Reply:
Then ignore me!